packages/server/src/main/resources/diagram-resource.ts

```diff
@@ -45,7 +43,15 @@ export class DiagramResource {
     if (width === undefined || height === undefined) {
       res.status(400).send('Both width and height must be defined');
     } else {
-      pdfMake.vfs = pdfFonts.pdfMake.vfs;
+      pdfMake.vfs = dejavuSans;
+      pdfMake.fonts = {
+        DejaVuSans: {
```

Comment:
I don't understand why we need a custom font here? is it not possible to solve the issue without a custom fonts?

Reply:
In any case I would prefer something closer to the default fonts of operating systems such as Helvetica or Arial (in case we can force this, potentially without adding them to the repo)

Reply:
> I don't understand why we need a custom font here? is it not possible to solve the issue without a custom fonts?

Apparently, PDF files have 14 standard fonts;  [Acrobat Reader Base Fonts](https://docs.oracle.com/cd/E96927_01/TSG/FAQ/What%20are%20the%2014%20base%20fonts%20distributed%20with%20Acroba.html)

These fonts are not supporting Unicode characters, arrows for our case.

Reply:
> In any case I would prefer something closer to the default fonts of operating systems such as Helvetica or Arial (in case we can force this, potentially without adding them to the repo)

We are using normal arrows for up and down, but long arrows for left and right. If we replace long arrows with normal ones, we can use Arial which supports normal arrows. Since Arial is also not supported in pdf, we have to use it as a custom font.

Reply:
Example: Standardized arrows & Arial font

![arial](https://user-images.githubusercontent.com/36473858/206923596-aa885f15-51c1-4aaf-becc-0e2c73cf1c60.png)

Reply:
Is there no other default PDF font which supports the arrow?

Reply:
I tried all of them, but no luck... 

[List of fonts supporting arrows](https://www.fileformat.info/info/unicode/char/2192/fontsupport.htm)

There is a library called [pdfmake-unicode](https://github.com/Landish/pdfmake-unicode) that provides Aerial GEO font that supports arrows but this library is not maintained anymore.